libnavigation-copilot/src/test/java/com/mapbox/navigation/copilot/MapboxCopilotImplTest.kt

```diff
@@ -1052,6 +1052,67 @@ class MapboxCopilotImplTest {
         assertEquals(1, attachments.size)
     }
 
+    @Test
+    fun `AttachmentMetadata created is set to startedAt`() {
+        val mockedMapboxNavigation = prepareBasicMockks()
+        prepareLifecycleOwnerMockk()
+        val mockedHistoryRecorder = mockk<MapboxHistoryRecorder>(relaxed = true)
+        every {
+            mockedMapboxNavigation.retrieveCopilotHistoryRecorder()
+        } returns mockedHistoryRecorder
+        val accessToken =
+            "pk.eyJ1IjoiY29waWxvdC10ZXN0LW93bmVyIiwiYSI6ImNpdDF3OHpoaTAwMDcyeXA5Y3Z0Nmk2dzEifQ." +
```

Comment:
what's this token? 😄  should it be something like "pk.test-token"

Reply:
All good, it's a fake token (actually it needs to be a "real" one so that we can retrieve the `owner`) 😉

Reply:
these tokens require manual look up though if there is a security audit, maybe make it a little more fake so that it doesn't need to be looked up?

Reply:
What do you mean by `make it a little more fake`?

Reply:
I think what @kmadsen means is that this token looks like a tangible token. Changing it to something like this won't raise any suspicion.
```kotlin
val accessToken = "pk.MapboxAccessToken"
```

Reply:
But if we do that we can't retrieve a "fake" valid `owner` from it.

Reply:
Is the "a" field in JWT payload also fake? 
Looks too legitimate right now, maybe replace it with a string "fake"?
And leave a comment for future security audits.

Reply:
Changed all occurrences of `accessToken` to `fakeAccessToken` and added a comment note for future security audits 👍

Reply:
Discussed internally with @azarovalex and we should be good now using `pk.eyJ1IjoiY29waWxvdC10ZXN0LW93bmVyIiwiYSI6ImZha2UifQ.8badf00d`. Thanks for the feedback 🙇‍♂️